spring-cloud-gcp-samples/spring-cloud-gcp-data-multi-sample/src/main/java/com/example/Trader.java

```diff
@@ -21,7 +21,7 @@
 import com.google.cloud.spring.data.spanner.core.mapping.Table;
 
 /** A sample entity. */
-@Table(name = "traders")
+@Table(name = "traders_repository")
```

Comment:
Why did this change?

Reply:
Why did it work before then? I don't remember ever seeing the `traders_repository` table.

Reply:
I checked the git history of Trader.java in [spanner repository sample](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/fc38c880b450cf555c36ceaa44eb45247cf70dcb/spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample). The table name used to be `traders`.

To run [multi databases sample](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/fc38c880b450cf555c36ceaa44eb45247cf70dcb/spring-cloud-gcp-samples/spring-cloud-gcp-data-multi-sample), you have to run spanner repository sample first to create all tables needed in the multi databases sample.

My guess is that the repository sample had been run with `traders` so that there's a `traders` table exists in the database and multi databases sample can use. However, when I run the tests with my own GCP project, it fails because no `traders` table exists.